.evg.yml

```diff
@@ -111,9 +113,122 @@ tasks:
     commands:
       - func: "generate sbom"
       - func: "upload sbom"
+      - func: "push SBOM Lite to Silk"
+      - func: "pull augmented SBOM from Silk"
       - func: "scan sbom"
 
+  - name: ssdlc-artifacts
+    run_on: ubuntu2204-small
+    depends_on:
+      - name: sbom
+        variant: code-quality-and-correctness
+    exec_timeout_secs: 300 # 5m
+    commands:
+      - func: "pull augmented SBOM from Silk"
+
+  - name: publish-augmented-sbom
+    run_on: ubuntu2204-small
+    git_tag_only: true
+    depends_on:
+      - name: "publish-maven"
+        variant: "release"
+      - name: ssdlc-artifacts
+        variant: "release"
+    exec_timeout_secs: 300 # 5m
+    commands:
+      - func: "publish augmented SBOM"
+
 functions:
+  "push SBOM Lite to Silk":
+    - command: shell.exec
+      type: test
+      params:
+        shell: bash
+        working_dir: mongo-jdbc-driver
+        script: |
+          ${PREPARE_SHELL}
+          cat << EOF > silkbomb.env
+          SILK_CLIENT_ID=${SILK_CLIENT_ID}
+          SILK_CLIENT_SECRET=${SILK_CLIENT_SECRET}
+          EOF
+
+          ls -lrt $SSDLC_DIR
+
+          echo "SBOM_LITE_NAME = $SBOM_LITE_NAME"
+
+          echo "-- Uploading initial SBOM Lite to Silk --"
+          docker run -i --platform="linux/amd64" --rm -v "$PWD":/pwd \
+          --env-file silkbomb.env \
+          artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:1.0 \
+          upload --silk-asset-group ${SILK_ASSET_GROUP} --sbom-in /pwd/artifacts/ssdlc/$SBOM_LITE_NAME
+          echo "-------------------------------"
+
+  "pull augmented SBOM from Silk":
+    - command: ec2.assume_role
+      params:
+        role_arn: ${assume_role_arn}
+        duration_seconds: 3600
+    - command: shell.exec
+      type: test
+      params:
+        shell: bash
+        working_dir: mongo-jdbc-driver
+        script: |
+          ${PREPARE_SHELL}
+          
+          mkdir -p $SSDLC_DIR
```

Comment:
Let's move this up in `export variables` so that this directory will always exist